README.md

```diff
@@ -54,7 +55,20 @@ While designing distributed systems are hard enough, testing them is even harder
 
 ### Programming Model
 - [Distributed Programming Model](http://web.cs.ucdavis.edu/~pandey/Research/Papers/icdcs01.pdf)
-- [PSync: a partially synchronous language for fault-tolerant distributed algorithms](http://www.di.ens.fr/~cezarad/popl16.pdf)
+- [PSync: a partially synchronous language for fault-tolerant distributed algorithms](http://www.di.ens.fr/~cezarad/popl16.pdf) Video: [Conference Video](https://www.youtube.com/watch?v=jxfq9_L9T1U&t=51s)
+- [Programming Models for Distributed Computing](http://heather.miller.am/teaching/cs7680/)
+- [Logic and Lattices for Distributed Programming](http://db.cs.berkeley.edu/papers/UCB-lattice-tr.pdf)
+
+### Verification in Distributed System
+- [Jepsen](https://github.com/jepsen-io/jepsen) A framework for distributed systems verification, with fault injection
+- [Verdi](http://verdi.uwplse.org/) A Framework for Implementing and Formally Verifying Distributed Systems [Paper](http://verdi.uwplse.org/verdi.pdf)
+
+### Videos of Conference Talks
```

Comment:
Given the presence of a 'Videos' section, it would be better to organize all videos under a single title and categorize them with tags such as '[Conference Talks]' for better clarity and organization.